clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/view/AmazonSubnetProvider.groovy

```diff
@@ -74,10 +77,14 @@ class AmazonSubnetProvider implements SubnetProvider<AmazonSubnet> {
     String target = null
     if (json) {
       def objectMapper = new ObjectMapper()
-      def metadata = objectMapper.readValue((String) json, Map.class)
-      metadata?.purpose
-      purpose = metadata?.purpose
-      target = metadata?.target
+      try {
+        def metadata = objectMapper.readValue((String) json, Map.class)
+        metadata?.purpose
```

Comment:
This line seems really odd (I know it's in the original code, just doesn't seem to serve any purpose). Can we remove it?